contrib/fuzz/docker_fuzzer.go

```diff
@@ -0,0 +1,78 @@
+// +build gofuzz
+
+/*
+   Copyright The containerd Authors.
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+       http://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+/*
+   This fuzzer is run continuously by OSS-fuzz.
+   It is stored in contrib/fuzz for organization,
+   but in order to execute it, it must be moved to
+   remotes/docker first. This is handled by OSS-fuzz.
+*/
+
+// nolint: golint
+package docker
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+)
+
+func FuzzFetcher(data []byte) int {
+	dataLen := len(data)
+	start := 0
```

Comment:
`start` appears to always be 0.  Why is this needed?

Reply:
> Why is this needed?

It isn't and has been removed.